lightyear/src/client/input.rs

```diff
@@ -330,3 +331,19 @@ fn prepare_input_message<P: Protocol>(
     input_manager.input_buffer.pop(interpolation_tick);
     // .pop(current_tick - (message_len + 1));
 }
+
+/// In host server mode, we don't buffer inputs (because there is no rollback) and we don't send
+/// inputs through the network, we just send directly to the server's InputEvents
+fn send_input_to_server<A: UserAction>(
```

Comment:
Maybe call this `*_unbuffered` or something so it doesn't sound like the standard code path

Reply:
You're right, thanks